cdap-app-fabric/src/main/java/io/cdap/cdap/app/store/Store.java

```diff
@@ -379,6 +380,16 @@ List<ProgramHistory> getRuns(Collection<ProgramReference> programs, ProgramRunSt
   List<ProgramSpecification> getDeletedProgramSpecifications(ApplicationReference appRef,
                                                              ApplicationSpecification specification);
 
+  /**
+   * updates source control meta of an application
+   *
+   * @param updateRequests List of {@link UpdateSourceControlMetaRequest}
+   *
+   * @throws IOException if scm meta update fails
+   */
+  void updateApplicationSourceControlMeta(List<UpdateSourceControlMetaRequest> updateRequests)
```

Comment:
List -> Collection

Reply:
any time a store method is added, it must be covered by new unit tests.

Reply:
Made the change to take a collection. And added unit test for the method.